webapp/src/components/Celebrities.vue

```diff
@@ -107,7 +110,9 @@
         operator: 'celebrity_detection',
         canvasRefreshInterval: undefined,
         timeseries: new Map(),
-        selectedLabel: ''
+        selectedLabel: '',
+        noResults: false,
```

Comment:
Not using this data prop, I was going to do something with it later, but I can't remember. Might remove this as part of the PR.